executor/ddl.go

```diff
@@ -77,12 +112,37 @@ func (e *DDLExec) Next(ctx context.Context, req *chunk.Chunk) (err error) {
 	e.done = true
 
 	// For each DDL, we should commit the previous transaction and create a new transaction.
-	// An exception is create local temporary table.
-	if s, ok := e.stmt.(*ast.CreateTableStmt); ok {
+	// Following cases are exceptions
+	var localTempTableNames []*model.TableInfo
```

Comment:
It is not an array of names and it is used for dropping, may be  localTempTableNames => localTempTablesToDrop is better?